website/docs/d/is_reservation.html.markdown

```diff
@@ -0,0 +1,159 @@
+---
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : reservation"
+description: |-
+  Manages IBM Cloud reservations.
+---
+
+# ibm_is_reservation
+Retrieve information of an existing reservation as a read only data source.
+
+**Note:** 
+VPC infrastructure services are a regional specific based endpoint, by default targets to `us-south`. Please make sure to target right region in the provider block as shown in the `provider.tf` file, if VPC service is created in region other than `us-south`.
```

Comment:
mention the link when its going to upstream. refer other resources

Reply:
There is no link yet, I will update once links are available